### PR TITLE
Home page

### LINKED ISSUE #1: Epic: Home page & Routing
# Home page

Feature: Home page

## Epic Description

Complete all features of the home

## List of Tasks (Complete in order)

- [ ] Create landing page
- [x] Complete basic routing
- [ ] Add featured categories on landing page

### PR DESCRIPTION
**Pull request by Pratham82**
# Project setup

[Epic: Home page #1](https://github.com/Pratham82/e-comm-react/issues/1)

## Changes made

- [x] Added assets
- [x] Added styles
- [x] Added pages routing in app
- [x] Added common components
- [x] Updated configs

## How does the solution address the problem

This PR will add basic routing and template pages

## Linked issues

Resolves #1

### CODE REVIEW

**Review by Asuraking1n — APPROVED** [review]
Except UI at some places everything looks fine to me

**Comment by Pratham82** [comment]
> Except UI at some places everything looks fine to me

I'll work on the individual pages, right now it's just a template.

File: src/pages/auth/index.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+export default function Login() {
+  return (
+    <section className="auth-section">
+      <div className="auth">
+        <ul className="tabs">
+          <li
+            id="login"
+            className="tabLinks active"
+            // onClick={() => console.log("clicked")}
```
Comment (sakshi006) [inline src/pages/auth/index.tsx:11]: Can remove commented code

File: src/pages/auth/index.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+          >
+            Login
+          </li>
+          <li
+            id="signup"
+            className="tabLinks"
+            // onClick="openTab(event, 'signup')"
```
Comment (sakshi006) [inline src/pages/auth/index.tsx:18]: This one also

File: src/pages/auth/index.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+              <button className="btn primary-filled-btn" type="button">
+                Login
+              </button>
+              <button className="btn primary-outlined-btn" type="button">
+                Logout
+              </button>
+            </div>
+          </div>
+          {/* <div id="signup" className="content">
+            <h4 className="h4 text-center">Sign Up</h4>
```
Comment (sakshi006) [inline src/pages/auth/index.tsx:58]: Can remove commented code

**Review by sakshi006 — APPROVED** [review]
Looks good to merge

File: src/pages/auth/index.tsx (hunk 1 above)
Comment (Pratham82) [reply src/pages/auth/index.tsx:11]: This was done in js so I had to convert it to rect, I'm going to work on auth in the upcoming tickets, thanks for the review

File: src/pages/auth/index.tsx (hunk 2 above)
Comment (Pratham82) [reply src/pages/auth/index.tsx:18]: This was done in js so I had to convert it to rect, I'm going to work on auth in the upcoming tickets, thanks for the review

**Pull request merged**